test/HttpClient.ts

```diff
@@ -0,0 +1,76 @@
+import test from "ava";
```

Comment:
```suggestion
import test from 'ava';
```
Looks like we need better ESLint rules :)